src/components/Keyboard.tsx

```diff
@@ -46,11 +46,13 @@ const Keyboard = ({
 
       if (letters.includes(letter)) {
         addLetter(letter)
+        event.preventDefault()
```

Comment:
No, not here. You _were_ right if the last branch was a pure `else`, but it is `else if`, but there is the implicit `else` branch where it shouldn't be called.

If you really want to move it out, you could do something like this

    let callPreventDefault = true
    if (...) {
    } else if (...) {
    } else if (...) {
    } else {
      callPreventDefault = false
    }
    callPreventDefault && event.preventDefault()

but it doesn't really save a lot of code or make it easier. I would probably consider this if there were 3+ shared lines of code between the branches, and even then it's probably still better style to just group those lines into a function so that the code shared between branches is again reduced to a single function call.